spaceship/lib/spaceship/spaceauth_runner.rb

```diff
@@ -49,6 +49,12 @@ def run
 
       yaml = cookies.to_yaml.gsub("\n", "\\n")
 
+      if Spaceship::Client::UserInterface.interactive? && agree("🙄 Should fastlane copy the cookie into your clipboard, so you can easily paste it? (y/n)", true)
+        require 'open3'
+        Open3.popen3('pbcopy') { |input, _, _| input << yaml }
```

Comment:
`pbcopy` is only a mac thing so this won't work if linux or windows 🤔 Are we able to get a cross platform solution? Or if not can we add a check specifically for macos?

Reply:
I noticed this as well and created an issue some time ago: https://github.com/fastlane/fastlane/issues/13897 Although for this PR it is probably out of scope.

But: I think the other "copy to clipboard" functionalities are hidden behind the `mac?` helper, which is a great idea to not suggest this can work on Linux or Windows.

Reply:
@joshdholtz @janpio So what's the solution here?

Reply:
Exactly what I wrote in my previous comment - make sure this code is only run when on Mac.

Reply:
@janpio I have added a check for `mac?`. Please review again.

Reply:
I confirmed that this doesn't get output on Windows. Someone else will have to confirm it does get output on Mac.

(Having tests for `spaceauth` would be neat...)